File: src/main/java/org/artembogomolova/demo/webapp/dao/repo/IAuthorityRepository.java
package org.artembogomolova.demo.webapp.dao.repo;

import org.artembogomolova.demo.webapp.domain.auth.Authority;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface IAuthorityRepository extends CrudRepository<Authority, Long> {

}

File: app/src/main/kotlin/org/artembogomolova/demo/webapp/main/dao/repo/IAuthorityRepository.kt
package org.artembogomolova.demo.webapp.main.dao.repo

import org.artembogomolova.demo.webapp.main.domain.auth.Authority
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface IAuthorityRepository : CrudRepository<Authority?, Long?>